projects/storefrontlib/src/cms-components/navigation/footer-navigation/scroll-to-top/scroll-to-top.component.scss

```diff
@@ -0,0 +1,34 @@
+.cx-main {
+  position: fixed;
+  bottom: 15px;
+  right: 15px;
+  opacity: 0;
+  transition: all 0.2s ease-in-out;
+}
+.cx-show-scrollTop {
+  opacity: 1;
+  transition: all 0.2s ease-in-out;
+}
+.cx-button-scroll-to-top {
+  border: none;
+  background: none;
+  padding: 0;
+}
+.cx-img-scroll-to-top {
+  display: block;
+  content: url('../../../../assets/scroll-to-top.png');
+}
+.cx-img-scroll-to-top-mobile {
+  display: none;
+  content: url('../../../../assets/scroll-to-top-mobile.png');
+}
+@media (max-width: 600px) {
```

Comment:
Use global breakpoint mechanism, eg. `@include media-breakpoint-up(lg) { ... }`

Reply:
Also, there is no need to create a separate block of code, just for mobile CSS. Simply, use the breakpoint mechanism as follows:
```CSS
.class {
  display: none;

  @include media-breakpoint-up(lg) {
    display: block
  }
}
```